### ISSUE TITLE
Library Deletion

### ISSUE THREAD

**Issue by Leunsensei** [tachiyomiorg/tachiyomi#8422]
### Describe your suggested feature

![Screenshot_2022-11-02-14-18-39-405_eu kanade tachiyomi](https://user-images.githubusercontent.com/98178685/199500419-9c058ec2-e893-4152-a3fe-e8f304694e1f.jpg)

The "from library" option should be automatically highlighted as that's the default reason anyone would click the delete option, having to click again is an unnecessary step.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have updated the app to version **[0.14.2](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

**Comment by arkon** [comment]
Not really, there were way too many people complaining that they accidentally removed things from the library.
